report/src/introduction/agile_approach.tex

```diff
@@ -1,29 +1,27 @@
 \section{Implementing Agile Project Management with Scrum}
 Developing a software product for a business can be a complex process.
-Incremental agile approaches have become popular to address this complexity, as they allow for faster development and adaptation to changes.\cite{sommervilleSoftwareEngineering2016}
-In incremental agile approaches, a system is developed as a series of increments, each adding functionality to the system.
-Between these increments, one gathers feedback from users to prioritize functionality for the next increment.
+Incremental agile approaches have become popular to address this complexity, as it allows for fast adaptation to changes in user needs. \cite{sommervilleSoftwareEngineering2016}
+In incremental-agile approaches, system development is considered a series of increments, each adding functionality to the system.
+Between these increments, one gathers feedback from stakeholders to prioritize functionality for the next increment.
 Therefore, each increment adds value to the user.
-By continuously getting feedback and working with these increments, the development process becomes cheaper and faster as a result of less time spent reworking the system.\cite{sommervilleSoftwareEngineering2016}
-This section will discuss how we are using an agile approach for our project, focusing specifically on the Scrum framework.
+By continuously getting feedback and working with these increments, the development process becomes cheaper and faster as less time is spent adapting to changes.\cite{sommervilleSoftwareEngineering2016}
```

Comment:
Ambivalent on this one. It's still correct, and I like the brevity. However, this _seems_ to indicate that adapting to changes is what we wish to avoid... and that's not correct.